jerry-core/config.h

```diff
@@ -37,9 +37,10 @@
  */
 #ifdef CONFIG_DISABLE_ES2015
 # define CONFIG_DISABLE_ES2015_ARROW_FUNCTION
-# define CONFIG_DISABLE_ES2015_CLASS
 # define CONFIG_DISABLE_ES2015_BUILTIN
+# define CONFIG_DISABLE_ES2015_CLASS
 # define CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
+# define CONFIG_DISABLE_ES2015_SHORTHAND_OBJ_LITERAL_NOTATION
```

Comment:
I would prefer a single guard for all object literal extensions. Perhaps `CONFIG_DISABLE_ES2015_OBJ_LITERAL` is enough.